Example/Podfile.lock

```diff
@@ -7,7 +7,7 @@ PODS:
   - Bolts/Tasks (1.1.5)
   - Gini-iOS-SDK (0.3.2):
     - Bolts (~> 1.1.0)
-  - GiniVision (3.0.4)
+  - GiniVision (3.0.5)
```

Comment:
So, then 3.0.5 should be the right one. However, again, it's an unrelated change.